### ISSUE TITLE
Error on Install: Failed to find amavis database schema

### ISSUE THREAD

**Issue by DaveDischord** [modoboa/modoboa-installer#413]
# Impacted versions

* Distribution: Ubuntu 
* Codename: Jammy
* Arch: 64 Bits
* Database: PostgreSQL

# Steps to reproduce
RUN 
`sudo python3 ./run.py --interactive example.com --debug 2>&1 | tee modoboaInstall_Output.txt`
I'm building a replacement to an existing email server, and LetsEncrypt is set to generate, but since the domain setup isn't complete, I did anticipate a failure on that bit.  That shouldn't have affected the Amavis install though.
# Full trace using --debug option or current behaviour
Using --debug and output results to file
# Expected behavior
Installation Successful
# Video/Screenshot link (optional)
[modoboaInstall_Output.txt](https://github.com/modoboa/modoboa-installer/files/8655658/modoboaInstall_Output.txt)

**Comment by DaveDischord** [comment]
LE Enabled?

**Comment by DaveDischord** [comment]
Quicker, but the same results.
[Output_noLE.txt](https://github.com/modoboa/modoboa-installer/files/8671806/Output_noLE.txt)

**Comment by DaveDischord** [comment]
Quicker, but same result
[Output_noLE.txt](https://github.com/modoboa/modoboa-installer/files/8671816/Output_noLE.txt)
.

**Comment by tonioo** [comment]
@DaveDischord Are you sure your installer local copy is up to date?

**Comment by DaveDischord** [comment]
Yes.  I used the instructions on [https://github.com/modoboa/modoboa-installer](https://github.com/modoboa/modoboa-installer) and ran 

`sudo ./run.py --interactive example.com`

The results are below 


```
Welcome to Modoboa installer!

Warning:
Before you start the installation, please make sure the following DNS records exist for domain 'example.com':
  mail IN A   <IP ADDRESS OF YOUR SERVER>
       IN MX  mail.example.com.

Your mail server will be installed with the following components:
modoboa automx amavis clamav dovecot nginx razor postfix postwhite spamassassin uwsgi radicale opendkim
Do you confirm? (Y/n) The process can be long, feel free to take a coffee and come back later ;)
Starting...
Generating new self-signed certificate
Installing amavis
Failed to find amavis database schema
```

**Comment by DaveDischord** [comment]
I went back to a pre-install snapshot and reran the installer with the --debug flag.  Same error, but I'm attaching the output file here if it helps
[modoboaInstall_SelfSigned.txt](https://github.com/modoboa/modoboa-installer/files/8672884/modoboaInstall_SelfSigned.txt)
.

**Comment by tonioo** [comment]
@DaveDischord With a fresh git clone of the installer repository or with an old one?

**Comment by DaveDischord** [comment]
It was a fresh git clone.  Even following the instructions straight from the README

**Comment by Infinit011** [comment]
Im having the same issue! Same machine as @DaveDischord

**Comment by Thedinney** [comment]
I'm having the same problem! Same machine  as @DaveDischord

**Comment by DaveDischord** [comment]
I ran through the fix you linked on a fresh git pull and still ran into an issue, though a different one.  This screen shot is without --debug.
![Screen Shot 05-17-22 at 02 20 PM](https://user-images.githubusercontent.com/14235646/168893397-23390364-d78c-4ecb-b451-78ad088736dd.PNG)
This screenshot is with --debug and hangs:
![Screen Shot 05-17-22 at 02 35 PM](https://user-images.githubusercontent.com/14235646/168895616-77a29208-6968-44de-a6c9-e8429096f0d6.PNG)

**Comment by tonioo** [comment]
@DaveDischord Just press the 'q' key in such case.

**Comment by DaveDischord** [comment]
OK.  I've re-cloned the installer and it seems like it's installing as
expected without any work-arounds.

**Comment by DaveDischord** [comment]
Fresh clone from git.
